assets/css/styleguide.scss

```diff
@@ -321,11 +325,27 @@ $site-top:           124px;
     top: 2em;
     bottom: 2em;
   }
-  padding: 1em $site-margins;
+  padding: 0 $site-margins $site-margins;
   border: 1px solid #eeeeee;
   .is-peripheral {
     opacity: .2;
   }
+
+  fieldset,
+  h6:first-child,
+  h6.usa-heading-alt:first-child {
+    margin-top: 3rem;
+  }
+
+  ul,
+  ol {
+    margin-bottom: 0;
+  }
+
+  > a {
+    margin-top: 3rem;
+    display: inline-block;
+  }
```

Comment:
@maya I had taken out the top margin on the preview box trying not to change the margin of the elements themselves, since some elements had margin tops to begin with. For the h6s they have a 4rem margin normally but the first one should be only 3rem. The > a was just for the link within that preview so that would only take that margin top when it was in a preview and not effect the a itself. ul and ol I wanted to take the margins out just for that box, but agreed that it should not effext the component itself. Will look into using a class this weekend.